Update delete-or-restore-mailboxes.md (#2883)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/recipients-in-exchange-online/delete-or-restore-mailboxes.md

When you delete a mailbox, Exchange Online retains the mailbox and all its contents until the deleted mailbox retention period expires, which is 30 days. After 30 days, the mailbox is permanently deleted and can't be recovered. The method for restoring a mailbox depends on whether the mailbox was deleted by deleting the user account or removing the Exchange Online license.

### To help understand the current status of a deleted mailbox

> To help you understand the current status of a recently deleted mailbox, we provide automated diagnostics in Microsoft 365 admin center. To launch the diagnostics, either type **Run Tests: Deleted Mailbox** when in the admin center Help panel or click [on this link](https://aka.ms/PillarDeletedMailBox) (this will take you directly to the Microsoft 365 admin center Help panel).

Reviewer: ```suggestion
To help you understand the current status of a recently deleted mailbox, we provide automated diagnostics in Microsoft 365 admin center. To launch the diagnostics, either type **Run Tests: Deleted Mailbox** when in the admin center Help panel or click [here](https://aka.ms/PillarDeletedMailBox) to open the Microsoft 365 admin center Help panel.
```

> To help you understand the current status of a recently deleted mailbox, we are providing an automated diagnostics in Microsoft 365 admin center. To launch the diagnostics, either type **Run Tests: Deleted Mailbox** when in admin center Help panel or click [on this link](https://aka.ms/PillarDeletedMailBox) (this will take you directly to the Microsoft 365 admin center Help panel).

Reviewer 2: ```suggestion
To help you understand the current status of a recently deleted mailbox, we provide automated diagnostics in Microsoft 365 admin center. To launch the diagnostics, either type **Run Tests: Deleted Mailbox** when in the admin center Help panel or click [on this link](https://aka.ms/PillarDeletedMailBox) (this will take you directly to the Microsoft 365 admin center Help panel).
```
